ui/components/app/edit-gas-display/edit-gas-display.component.js

```diff
@@ -110,7 +110,11 @@ export default function EditGasDisplay({
   );
 
   let warningMessage;
-  if (new BigNumber(gasLimit).lessThan(new BigNumber(properGasLimit))) {
+  if (
+    gasLimit &&
```

Comment:
Is `gasLimit` a number or a hex value here? Just wondering whether it would make more sense to say `if (Boolean(gasLimit) && Boolean(properGasLimit) && ...)` here.

Reply:
@mcmire `gasLimit` is a number. I tried with this changes that you suggested here `(Boolean(gasLimit) && Boolean(properGasLimit) && ...)`, unit test and coverage are passed also.
@danjm Can you please look at this and comment what do you think about it? I can add this changes if you agree with this.

Reply:
@mcmire for which case does the `Boolean` call make a difference?

Reply:
`Boolean(...)` would check if the value in question is null or undefined. Without that, if `gasLimit` is 0, but is less than `properGasLimit`, then we wouldn't show a warning. Seems like we might not want to do that.

Reply:
`Boolean(0) === false` right?

So do you mean that we need to do something like `gasLimit || gasLimit === 0`... or, what if we do `parseInt(gasLimit) !== NaN`?

Reply:
I just pushed a commit that I think resolves the concern

Reply:
Yup, that looks good!